author: bearinmindcat <bearincrypto@gmail.com>

---
--- a/chrome/browser/safe_browsing/BUILD.gn
+++ b/chrome/browser/safe_browsing/BUILD.gn
@@ -115,6 +115,11 @@
       deps += [ "//chromeos/ash/components/system" ]
     }
   }
+
+  deps = []
+  public = []
+  sources = []
+  allow_circular_includes_from = []
 
   # Note: is_android is not equivalent to safe_browsing_mode == 2.
   # Sources and dependencies added to this block must not depend on anything
