author: bearinmindcat <bearincrypto@gmail.com>

--- a/components/enterprise/connectors/core/cloud_content_scanning/cloud_binary_upload_service_base.cc
+++ b/components/enterprise/connectors/core/cloud_content_scanning/cloud_binary_upload_service_base.cc
@@ -357,17 +357,6 @@ void CloudBinaryUploadServiceBase::LogResponseDebugInfo(
     ScanRequestUploadResult result,
     BinaryUploadRequest* request,
     const ContentAnalysisResponse& response) {
-#if !BUILDFLAG(IS_IOS)
-  safe_browsing::WebUIContentInfoSingleton::GetInstance()
-      ->AddToDeepScanRequests(request->per_profile_request(),
-                              request->access_token(), upload_info,
-                              request->GetUrlWithParams().spec(),
-                              request->content_analysis_request());
-  safe_browsing::WebUIContentInfoSingleton::GetInstance()
-      ->AddToDeepScanResponses(active_tokens_[request->id()],
-                               ScanRequestUploadResultToString(result),
-                               response);
-#endif
 }

 void CloudBinaryUploadServiceBase::IsAuthorized(const GURL& url,
