author: bearinmindcat <bearincrypto@gmail.com>

diff --git a/chrome/browser/password_manager/android/all_passwords_bottom_sheet_controller.cc b/chrome/browser/password_manager/android/all_passwords_bottom_sheet_controller.cc
index 0593725dee8ba..432446c2c5ba0 100644
--- a/chrome/browser/password_manager/android/all_passwords_bottom_sheet_controller.cc
+++ b/chrome/browser/password_manager/android/all_passwords_bottom_sheet_controller.cc
@@ -10,6 +10,7 @@
 #include "chrome/browser/password_manager/chrome_password_manager_client.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/safe_browsing/chrome_password_reuse_detection_manager_client.h"
+#include "components/safe_browsing/buildflags.h"
 #include "chrome/browser/ui/android/passwords/all_passwords_bottom_sheet_view.h"
 #include "chrome/browser/ui/android/passwords/all_passwords_bottom_sheet_view_impl.h"
 #include "components/device_reauth/device_authenticator.h"
@@ -73,8 +74,10 @@ AllPasswordsBottomSheetController::AllPasswordsBottomSheetController(
           focused_frame);
   driver_ = driver->AsWeakPtr();
   client_ = ChromePasswordManagerClient::FromWebContents(web_contents_);
+#if BUILDFLAG(SAFE_BROWSING_AVAILABLE)
   password_reuse_detection_manager_client_ =
       ChromePasswordReuseDetectionManagerClient::FromWebContents(web_contents_);
+#endif
 }
 
 AllPasswordsBottomSheetController::~AllPasswordsBottomSheetController() {
@@ -195,7 +198,9 @@ void AllPasswordsBottomSheetController::FillPassword(
     return;
   }
   driver_->FillIntoFocusedField(true, password);
-  password_reuse_detection_manager_client_->OnPasswordSelected(password);
+  if (password_reuse_detection_manager_client_) {
+    password_reuse_detection_manager_client_->OnPasswordSelected(password);
+  }
 }
 
 void AllPasswordsBottomSheetController::OnResultFromAllStoresReceived(
diff --git a/chrome/browser/ui/android/safe_browsing/password_reuse_dialog_view_android.cc b/chrome/browser/ui/android/safe_browsing/password_reuse_dialog_view_android.cc
index c4da8c4d89e6a..8af259bd6a74b 100644
--- a/chrome/browser/ui/android/safe_browsing/password_reuse_dialog_view_android.cc
+++ b/chrome/browser/ui/android/safe_browsing/password_reuse_dialog_view_android.cc
@@ -27,30 +27,13 @@ PasswordReuseDialogViewAndroid::~PasswordReuseDialogViewAndroid() {
       base::android::AttachCurrentThread(), java_object_);
 }
 
-void PasswordReuseDialogViewAndroid::Show(ui::WindowAndroid* window_android) {
-  JNIEnv* env = base::android::AttachCurrentThread();
-  java_object_.Reset(Java_SafeBrowsingPasswordReuseDialogBridge_create(
-      env, window_android->GetJavaObject(), reinterpret_cast<intptr_t>(this)));
+void PasswordReuseDialogViewAndroid::Show(ui::WindowAndroid* window_android) {}
 
-  std::u16string warning_detail_text = controller_->GetWarningDetailText();
+void PasswordReuseDialogViewAndroid::CheckPasswords(JNIEnv* env) {}
 
-  Java_SafeBrowsingPasswordReuseDialogBridge_showDialog(
-      env, java_object_, controller_->GetTitle(), warning_detail_text,
-      controller_->GetPrimaryButtonText(),
-      controller_->GetSecondaryButtonText());
-}
-
-void PasswordReuseDialogViewAndroid::CheckPasswords(JNIEnv* env) {
-  controller_->ShowCheckPasswords();
-}
+void PasswordReuseDialogViewAndroid::Ignore(JNIEnv* env) {}
 
-void PasswordReuseDialogViewAndroid::Ignore(JNIEnv* env) {
-  controller_->IgnoreDialog();
-}
-
-void PasswordReuseDialogViewAndroid::Close(JNIEnv* env) {
-  controller_->CloseDialog();
-}
+void PasswordReuseDialogViewAndroid::Close(JNIEnv* env) {}
 
 }  // namespace safe_browsing
 
diff --git a/chrome/browser/ui/android/safe_browsing/password_reuse_dialog_view_android.h b/chrome/browser/ui/android/safe_browsing/password_reuse_dialog_view_android.h
index 036139c757bd0..f9b2bbd572e95 100644
--- a/chrome/browser/ui/android/safe_browsing/password_reuse_dialog_view_android.h
+++ b/chrome/browser/ui/android/safe_browsing/password_reuse_dialog_view_android.h
@@ -45,7 +45,7 @@ class PasswordReuseDialogViewAndroid {
  private:
   // The controller which owns this dialog and handles the dialog events.
   // |controller_| owns |this|.
-  raw_ptr<PasswordReuseControllerAndroid> controller_;
+  [[maybe_unused]] raw_ptr<PasswordReuseControllerAndroid> controller_;
 
   // The corresponding java object.
   base::android::ScopedJavaGlobalRef<jobject> java_object_;
