author: bearinmindcat <bearincrypto@gmail.com>

diff --git a/third_party/android_deps/autorolled/BUILD.gn b/third_party/android_deps/autorolled/BUILD.gn
--- a/third_party/android_deps/autorolled/BUILD.gn
+++ b/third_party/android_deps/autorolled/BUILD.gn
@@ -196,7 +196,6 @@
         deps = [
           "$google_play_services_package:google_play_services_base_java",
           "$google_play_services_package:google_play_services_basement_java",
-          "$google_play_services_package:google_play_services_flags_java",
           "$google_play_services_package:google_play_services_tasks_java",
           "//third_party/androidx:androidx_core_core_java",
           "//third_party/androidx:androidx_mediarouter_mediarouter_java",
@@ -296,7 +295,6 @@
         deps = [
           "$google_play_services_package:google_play_services_base_java",
           "$google_play_services_package:google_play_services_basement_java",
-          "$google_play_services_package:google_play_services_stats_java",
           "$google_play_services_package:google_play_services_tasks_java",
           "//third_party/androidx:androidx_collection_collection_java",
           "//third_party/androidx:androidx_core_core_java",
@@ -577,7 +575,6 @@
           "$google_play_services_package:google_play_services_base_java",
           "$google_play_services_package:google_play_services_basement_java",
           "$google_play_services_package:google_play_services_cloud_messaging_java",
-          "$google_play_services_package:google_play_services_stats_java",
           "$google_play_services_package:google_play_services_tasks_java",
           "//third_party/android_deps:com_google_errorprone_error_prone_annotations_java",
           "//third_party/androidx:androidx_annotation_annotation_java",
