author: bearinmindcat <bearincrypto@gmail.com>

diff --git a/components/signin/internal/identity_manager/primary_account_manager.cc b/components/signin/internal/identity_manager/primary_account_manager.cc
--- a/components/signin/internal/identity_manager/primary_account_manager.cc
+++ b/components/signin/internal/identity_manager/primary_account_manager.cc
@@ -103,8 +103,8 @@
   }
 }
 
-void LogPrimaryAccountPrefsOnInitialize(const std::string& pref_account_id,
-                                        bool pref_consented_to_sync) {
+[[maybe_unused]] void LogPrimaryAccountPrefsOnInitialize(
+    const std::string& pref_account_id, bool pref_consented_to_sync) {
   if (pref_account_id.empty()) {
     base::UmaHistogramEnumeration(
         "Signin.PAMInitialize.PrimaryAccountPrefs",
