From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 8 Apr 2020 20:48:17 -0400
Subject: [PATCH] enable -ftrivial-auto-var-init=zero

---
 build/config/compiler/BUILD.gn | 8 ++-----
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index e1da114..d14b8a2 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -3195,12 +3195,8 @@ if (is_android) {
 # Initialize all variables on the stack if needed.
 config("default_init_stack_vars") {
   cflags = []
-  if (init_stack_vars && is_clang && !using_sanitizer) {
-    if (init_stack_vars_zero) {
-      cflags += [ "-ftrivial-auto-var-init=zero" ]
-    } else {
-      cflags += [ "-ftrivial-auto-var-init=pattern" ]
-    }
+  if (is_clang && !using_sanitizer) {
+    cflags += [ "-ftrivial-auto-var-init=zero" ]
   }
 }
 
