From 560f1408b7556b8211458d81f9b735d02a30bc4b Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 23 Dec 2020 06:00:50 -0500
Subject: [PATCH 43/78] enable split cache by default
author: bearinmindcat <bearincrypto@gmail.com>
date: Tue, 22 Jul 2026
---
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -197,13 +197,13 @@
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kSplitCacheByNetworkIsolationKey,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kSplitCodeCacheByNetworkIsolationKey,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kSplitHostCacheByNetworkAnonymizationKey,
              base::FEATURE_ENABLED_BY_DEFAULT);
