description: modify GN flags and pruning list
author: Wengling Chen <feiyu2817@gmail.com>
author: bearinmindcat <bearincrypto@gmail.com>
date: Fri, 10 Jul 2026
---
diff --git a/ungoogled-chromium/flags.gn b/ungoogled-chromium/flags.gn
index 8394b02..40909d5 100644
--- a/ungoogled-chromium/flags.gn
+++ b/ungoogled-chromium/flags.gn
@@ -7,7 +7,6 @@ enable_remoting=false
 enable_reporting=false
 enable_service_discovery=false
 enable_widevine=true
-exclude_unwind_tables=true
 google_api_key=""
 google_default_client_id=""
 google_default_client_secret=""
diff --git a/ungoogled-chromium/pruning.list b/ungoogled-chromium/pruning.list
index 3382d5b..8c37145 100644
--- a/ungoogled-chromium/pruning.list
+++ b/ungoogled-chromium/pruning.list
@@ -2130,7 +2130,6 @@ third_party/angle/third_party/glmark2/src/data/models/asteroid-low.3ds
 third_party/angle/third_party/glmark2/src/data/models/cat.3ds
 third_party/angle/third_party/glmark2/src/data/models/cube.3ds
 third_party/angle/third_party/glmark2/src/data/models/horse.3ds
-third_party/angle/third_party/r8/custom_d8.jar
 third_party/aria-practices/src/common/utility/aria-taxonomy.xmi.zargo
 third_party/aria-practices/src/content/patterns/slider/examples/images/max-arrow.paint
 third_party/blink/renderer/bindings/core/v8/serialization/fuzz_corpus/blink-0bb59816-x86
@@ -12132,7 +12131,6 @@ third_party/nasm/travis/test/xpaste.bin.t
 third_party/nasm/travis/test/zerobyte.bin.t
 third_party/nlohmann_json/src/tests/reports/2016-08-29-fuzz/fuzz.tiff
 third_party/nlohmann_json/src/tests/reports/2016-10-02-fuzz/fuzz.tiff
-third_party/node/node_modules/node_modules.tar.gz
 third_party/oak/src/.xtask_bash_completion.zwc
 third_party/oak/src/cc/crypto/tink/signature/testdata/public_keyset
 third_party/oak/src/cc/crypto/tink/signature/testdata/signature
@@ -12279,10 +12277,6 @@ third_party/pyelftools/test/testfiles_for_unittests/lib_with_two_dynstr_sections_reversed.so.1.elf
 third_party/pyelftools/test/testfiles_for_unittests/sample_exe64.elf
 third_party/pyelftools/test/testfiles_for_unittests/simple_gcc.elf.arm
 third_party/pywebsocket3/src/test/cert/client_cert.p12
-third_party/quic_trace/src/examples/example.json.gz
-third_party/quic_trace/src/examples/example.qtr
-third_party/r8/custom_d8.jar
-third_party/r8/custom_r8.jar
 third_party/rust/chromium_crates_io/vendor/fdeflate-v0_3/tests/input-chunking-sensitivity-example1.zz
 third_party/rust/chromium_crates_io/vendor/fdeflate-v0_3/tests/input-chunking-sensitivity-example2.zz
 third_party/rust/chromium_crates_io/vendor/fdeflate-v0_3/tests/input-chunking-sensitivity-example3.zz
